docs/releases/4.2.txt

```diff
@@ -35,6 +35,8 @@ Minor features
 * The light or dark color theme of the admin can now be toggled in the UI, as
   well as being set to follow the system setting.
 
+* The "Chosen" column of a ``FilteredSelectMultiple`` widget is now filterable.
```

Comment:
`filter_horizontal` and `filter_vertical` are both linkable (I think) and are the user-facing API here. I think mentioning that, rather than `FilteredSelectMultiple` is the way to go.